plugins/GSdx/GSdx.cpp

```diff
@@ -276,9 +276,10 @@ void GSdxApp::Init()
 	m_gs_offset_hack.push_back(GSSetting(2,  "Special", "Texture"));
 	m_gs_offset_hack.push_back(GSSetting(3,  "Special", "Texture - aggressive"));
 
+	m_gs_hw_mipmapping.push_back(GSSetting(-1, "Automatic", "Default"));
 	m_gs_hw_mipmapping.push_back(GSSetting(0, "Off", ""));
 	m_gs_hw_mipmapping.push_back(GSSetting(1, "Basic", "Fast"));
-	m_gs_hw_mipmapping.push_back(GSSetting(2, "Full", "Very Slow - Not Recommended"));
+	m_gs_hw_mipmapping.push_back(GSSetting(2, "Full", "Slow - Not Recommended"));
```

Comment:
I think you can also use the enum values here instead of integer constants.

Reply:
I think it's fine now. That will make the code just bigger. Plus there are other options that can be converted with enums as well if we look at it xD

Reply:
I'm just suggesting to use the already defined ``HWMipmap::Off`` instead of ``0`` at this place and similar for all other values. I'm not sure why you think it'll make the code bigger, you'll still have the same line of changes.

Reply:
Ah I thought you meant as in also creating a new enum as well. I'll update it in a min.